src/main/java/de/zalando/aruha/nakadi/service/subscription/ExactWeightRebalancer.java

```diff
@@ -70,6 +70,12 @@
     }
 
     static int[] splitByWeight(final int itemCount, final int[] weigths) {
+        if (itemCount < weigths.length) {
```

Comment:
What if I there are 2 partitions and 3 sessions? Isn't it a valid result to have `[1, 1, 0]` ?